modules/apigee_edge_teams/src/Plugin/EntityReferenceSelection/TeamMembersSelection.php

```diff
@@ -169,7 +170,13 @@ public function buildConfigurationForm(array $form, FormStateInterface $form_sta
    * {@inheritdoc}
    */
   protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS') {
-    $query = parent::buildEntityQuery($match, $match_operator);
+    // Do not pass $match so that it does not add the "name" field to the query,
+    // as we will search by email instead.
+    $query = parent::buildEntityQuery(NULL, $match_operator);
+    if (isset($match)) {
```

Comment:
It is badly documented but `$match` could be `null` or a `string`. If it could be a string then it could be an empty string as well so I would handle that case here by calling `trim` on the input.

Reply:
Agreed.